### DISCUSSION TITLE
Is it possible to downgrade a specific language parser?

### DISCUSSION THREAD

**Discussion by joshuahicks** [nvim-treesitter/nvim-treesitter#4728] [Q&A] [answered]
Hi, 

The current state of the `http` parser for this project is broken. There is an [open issue](https://github.com/rest-nvim/tree-sitter-http/issues/11) on the `tree-sitter-http` repo. I'm expecting it to get fixed, but in the meantime, my treesitter plugin breaks with `*.http` files. 

I'd like to downgrade the version of the http parser, and I'm not sure how.

Is this possible? If so, how could I do this?

**Comment by clason** [comment, marked as answer]
Yes, just change the commit hash in the `lockfile.json`. 

This specific issue looks like a config error, not a broken parser -- you must have some outdated queries in your runtime path. Make sure nvim-treesitter is fully up-to-date, and investigate all plugins that ship their own queries (see the pinned issue).

**Reply by joshuahicks** [reply]
Thank you 🙏 I'll take a look at my config closer, and I'll take a look at the pinned issue.
